types/.eslintrc.json

```diff
@@ -36,6 +36,9 @@
         "@typescript-eslint/explicit-function-return-type": "off",
         "@typescript-eslint/no-unused-vars": "off",
         "@typescript-eslint/no-non-null-assertion": "off"
+      },
+      "globals": {
+        "NodeJS": "readonly"
```

Comment:
Why do we need this?

Reply:
This is because of reference of `NodeJS.ErrnoException` which in one case breaks `tsd` and one other breaks `eslint`.